scripts/world-time-status.test.sh

```
@@ -0,0 +1,92 @@
+  .worldTime.year == 1993 and
+  .worldTime.month == 7 and
+  .worldTime.day == 9 and
+  .worldTime.daysSurvived == 12 and
+  .worldTime.worldAgeMinutes == 18030 and
+  (.worldCreatedAt | type == "string") and
+  (.worldAgeSeconds >= 0)
+' "$TMP_DIR/status.json" >/dev/null
+
+# Telemetry newer than this activation but older than the freshness limit is stale.
+touch -d '10 minutes ago' "$TMP_DIR/mods/zomboid-arm-world-telemetry/world-time.txt"
+export SYSTEMD_ACTIVE_ENTER="$(date -u -d '20 minutes ago' '+%Y-%m-%d %H:%M:%S UTC')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)